core/templates/dev/head/css/oppia.css

```diff
@@ -2988,3 +2988,333 @@ md-card.preview-conversation-skin-supplemental-card {
+  position: absolute;
+  display: inline-block;
+  bottom: -2.3em;
+}
+
+.otter-speech-card {
+  width: 50%;
+  float: right;
+}
+
+.otter-speech-bubble {
+  position: relative;
+  padding: 50px 15px;
+  margin: 1em 0 3em;
+  text-align: center;
+  background: #fff;
+  border-radius: 5px;
+  border-left: none;
+}
+
+.otter-speech-bubble:before {
+  content: "";
+  position: absolute;
+  z-index: 10;
+  bottom: -1.7em;
+  left: 1em;
+  width: 50px;
+  height: 30px;
+  background: #fff;
+  display: block;
+  border-bottom-right-radius: 80px 50px;
+}
+
+.otter-speech-bubble:after {
+  content: "";
+  position: absolute;
+  z-index: 10;
+  bottom: -1.7em;
+  left: 1em;
+  width: 20px;
+  height: 30px;
+  background: #eee;
+  border-bottom-right-radius: 40px 50px;
+  display: block;
+}
+
+.about-header h1 {
+  color: #fff;
+  font-weight: bold;
+  font-size: 3em;
+  text-shadow: 1px 2px 3px #38b1a5;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+}
+
+.about-extra-container h1,
+.about-extra-container h2,
+.about-extra-container h3,
+.about-extra-container h4,
+.about-extra-container h5,
+.about-extra-container h6 {
+  text-align: center;
+}
+
+.about-extra-content p {
+  margin: 1.5em 0;
+}
+
+.about-card-content-wide {
+  width: 90%;
+  padding: 8% 0;
+  margin: 0 auto;
+}
+
+.about-card-content-narrow {
+  width: 70%;
+}
+
+.create-three-cols-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 0 8% 0;
+}
+
+.create-three-cols-feature {
+  text-align: center;
+}
+
+.create-three-cols-layout {
+  width: calc((100% - 20px) / 3);
+  padding: 10px;
+  margin-right: 10px;
+}
+
+.create-three-cols-layout:last-of-type {
+  margin-right: 0;
+}
+
+.letter-groups {
+  margin: 0 0 10% 0;
+}
+
+.about-extra-info {
+  width: 100%;
+  color: #fff;
+}
+
+.about-otter-buttons {
+  background-color: #009788;
+}
+
+.about-get-involved {
+  background-color: #009788;
+}
+
+.new-explorations {
+  background-color: #16DEB7;
+}
+
+.start-creating {
+  background-color: #0d8885;
+}
+
+.about-tab-content {
+  display: none;
+}
+
+.about-visible-content {
+  position: relative;
+  opacity: 1;
+  display: block;
+  transition: opacity .35s ease;
+}
+
+.inline-links {
+  text-decoration: none;
+}
+
+.about-content p {
+  margin: 1.5em 0;
+}
+
+.about-content ol,
+.about-content ul {
+  padding: 0;
+}
+
+.about-content h2 {
+  text-align: center;
+}
+
+.otter-content h3 {
+  text-align: center;
+}
+
+.about-tab-links {
+  text-align: center;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.about-tab-links li {
+  display: inline-block;
+  cursor: pointer;
+  width: calc(100% / 3);
+  border-right: 1px solid rgba(0,0,0,0.1);
+  padding: 10px;
+  margin-bottom: 0;
+}
+
+.about-tab-links li:last-of-type {
+  border-right: none;
+}
+
+.about-tab-links .about-active-tab {
+  border-bottom: 2px solid #009688;
+}
+
+.about-active-tab a {
+  text-decoration: none;
+}
+
+.about-page-otter {
+  width: 250px;
+}
+
+.nav-creators ul li{
```

Comment:
This is an example of a class name that feels a little bit too broad. When I see it I think of something to do with creators in the site navbar.